src/Simulation/Native/src/simulator/factory.cpp

```diff
@@ -31,7 +32,8 @@ namespace Microsoft
   {
     namespace Simulator
     {
-      mutex_type _mutex;
```

Comment:
@dbwz8 This code used to use the #define that resolves to `omp_mutex` when OpenMP support is enabled. However, this code doesn't interact with any other OpenMP code/classes, so I think it should be safe to use the std class instead, especially since we compile with C++17 across all platforms now. Please let me know if this assumption is safe, and if not we can look at other solutions!

Reply:
Looks OK to me.